cmd/state/internal/cmdtree/invite.go

```diff
@@ -31,13 +34,18 @@ func newInviteCommand(prime *primer.Values) *captain.Command {
 		},
 		[]*captain.Argument{
 			{
-				Name:        "email1,[email2,..]",
+				Name:        "email1[,email2,..]",
 				Description: locale.Tl("invite_arg_emails", "Email addresses to send the invitations to"),
 				Required:    true,
 				Value:       &params.EmailList,
 			},
 		},
-		func(ccmd *captain.Command, _ []string) error {
+		func(ccmd *captain.Command, args []string) error {
+			if len(args) > 1 {
```

Comment:
We should maybe also guard against people giving both comma-separated emails as well as space separated (I'd just error out if they do this).